migration/mule/dataweave-with-flowreflookup/README.md

```diff
@@ -0,0 +1,56 @@
+# Dataweave-with-flowreflookup
+
+This example shows you how to utilize the [Salesforce REST Connector](https://store.wso2.com/store/assets/esbconnector/details/43e44763-0d73-4ab3-8ae9-d6f73532d164) to create sObjects from data stored in csv format and how to use sequence references to call another sequence.
+
+### Assumptions ###
+
+This document assumes that you are familiar with WSO2 EI and the 
+[Integration Studio interface](https://ei.docs.wso2.com/en/latest/micro-integrator/overview/quick-start-guide/). To 
+increase your familiarity with Integration Studio, consider completing one or more 
+[WSO2 EI Tutorials](https://ei.docs.wso2.com/en/latest/micro-integrator/use-cases/integration-use-cases/).
+
+### Example Use Case
+
+In this example, Multiple details of an account are stored in a csv file and it will be read using file connector and transform by referring to another sequence(Which invokes a script mediator to change specific region field) and create the sObject in salesforce using salesforcerest connector.
```

Comment:
The strength we have here is - lookup `LookupSalesRegionSequence` beforehand and assign the result to a variable and accessing that in the mediation flow. Shall we highlight that in the readme? 

It is true that we cannot lookup something from current datamapper impl, but we have above capability.

Reply:
Fixed in f730809f93872d6f312143b05d2121172452b449